build/container/Dockerfile

```diff
@@ -59,6 +61,9 @@ ENV NRIA_IS_SECURE_FORWARD_ONLY true
 #################################
 FROM forwarder AS k8s-events-forwarder
 
+RUN apk add --no-cache \
+        curl=$curl_version
+
```

Comment:
Does this work if run after the `USER nri-agent` directive? I'd expect it to fail due to lack of permissions

Reply:
nice catch!

Reply:
interesting, what does `Testing / Building containerized agent` do then 🤔  - https://github.com/newrelic/infrastructure-agent/pull/530/checks?check_run_id=2794152021

Reply:
I wonder the same thing, because apparently the forwarder steps have been skipped: Only `core` and `base` appear in the build log 😕

Reply:
I think this is the cause: There's a `--target` flag that might not be in use in that pipeline: https://github.com/newrelic/infrastructure-agent/blob/0c086311fb9b808c0224acc6244db33c6a1829cc/build/container/Makefile#L80

Reply:
moved that into forwarder image + added more tests in the pipeline